src/containers/Tenant/Query/SavedQueries/SavedQueries.tsx

```diff
@@ -154,7 +160,12 @@ export const SavedQueries = ({changeUserInput}: SavedQueriesProps) => {
                         settings={QUERY_TABLE_SETTINGS}
                         emptyDataMessage={i18n(filter ? 'history.empty-search' : 'saved.empty')}
                         rowClassName={() => b('row')}
-                        onRowClick={(row) => onQueryClick(row.body, row.name)}
+                        onRowClick={async (row) =>
```

Comment:
according to gravity-ui/react-data-table onRowClick is sync function
Does passing async function have purpose here?